src/sentry/tasks/integrations/migrate_repo.py

```diff
@@ -12,6 +13,7 @@
     queue="integrations",
     default_retry_delay=60 * 5,
     max_retries=5,
+    silo_mode=SiloMode.REGION,
```

Comment:
Does this only get spawned in the regions?

Reply:
Good question. From its current usages it actually looks like it's primarily geared for the control silo so I definitely mislabeled this.